concent-builder/configure-and-run-tests.yml

```diff
@@ -0,0 +1,55 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+    - ../containers/versions.yml
+    - "{{ deployment_values }}/var.yml"
+    - "{{ deployment_values }}/var-{{ cluster }}.yml"
+  tasks:
+    - become:      yes
+      become_user: jenkins
+      block:
+        - name: Copy concent and virtualmachine for deployment build
```

Comment:
`virtualmachine` -> `virtual machine`

Reply:
I corrected the name, this is virtual environment.